### ISSUE TITLE
Unable to run any project using MeshImport dll

### ISSUE THREAD

**Issue by Scylardor** [bagobor/opengl33_dev_cookbook_2013#2]
I'm trying to run the Dual Quaternion Skinning example project, and while it compiles well it fails at runtime to load the MeshImport_x86.dll. All projects using this DLL fail to use it

The [LoadLibraryA function call](https://github.com/bagobor/opengl33_dev_cookbook_2013/blob/master/Chapter8/DualQuaternionSkinning/MeshImport.cpp#L36) returns error code 126, a.k.a. "The specified module cannot be found."
It's a pretty cryptic error so I'm at a loss here.

Did anyone have this issue and knows how to fix ? I tried a number of things like moving the DLL next to the exe, but nothing worked. 

I'm using VS2019 on Windows 10, but using another, older version of VS doesn't help.

**Comment by mmmovania** [comment]
Seems like an x86 dll being used in an x64 build. Can you try to use the x86 compilation and see if it works?

**Comment by Scylardor** [comment]
Yes, unfortunately it has the same issue.
I don't know what could go wrong here, according to me it's supposed to work.
Maybe this DLL just isn't compatible with Windows 10 ?
I'll investigate a bit more and update if I find something interesting.

**Comment by Scylardor** [comment]
Ok so huh... The good news is I fixed my problem.
I had to download another version of the MeshImport and Ezm MeshImport DLLs that I found here : 
https://github.com/kmammou/meshimport/tree/master/bin/win32

For some reason, the version provided by this repo doesn't work on my machine, but those from there did the trick.
The bad news is that I don't know why it wasn't working in the first place.
Also, I had to move the two DLLs next to the executable file.

Thanks for the heads-up closing this issue.
